app/models/comments.models.js

```diff
@@ -21,6 +21,9 @@ exports.insertNewComment = (article_id, username, body) => {
       [body, username, article_id]
     )
     .then((comment) => {
+      if (!comment.rows.length) {
+        return Promise.reject({ status: 404, msg: "No comments found" });
+      }
```

Comment:
I know this isn't for the same endpoint but I'm not sure you need this if statement, the _articleID_ is a foreign key reference so if it can't be found psql should throw an error for you I think